### DISCUSSION TITLE
avoid indenting with spaces

### DISCUSSION THREAD

**Discussion by Belamat** [lukas-reineke/indent-blankline.nvim#366] [Q&A] [answered]
Hi Lukas, thanks for your contribution, your plugin works pretty nice : )

My question is how can I avoid indenting with spaces on the first characters?

example

if(something){
| | | | | | | | | WORDHERE=false;
| blabla2=true;
}

im trying to avoid the | | | | | | | | | before the WORDHERE, and also the spaces at the begging of a empy line

could this be achieved?

**Comment by lukas-reineke** [comment, marked as answer]
I am not 100% sure I understand, but I'll try to answer.

> im trying to avoid the | | | | | | | | | before the WORDHERE

You can use `:help g:indent_blankline_max_indent_increase` to only increase the indent level by 1, regardless of how deep a line is indented. Does that help?

> the spaces at the begging of a empy line

You can turn off the first indent guide with `:help g:indent_blankline_show_first_indent_level`

**Reply by Belamat** [reply]
thanks for the reply, 
it works but not fully ,it indents if the previous line doesnt have a space on it. not shure, i dont want to complicate it since i understand the not clearness of my question,

resuming it, it would be

could I avoid indenting with "space" and using just TAB for that? (perhaps thats on the vim default settings and not to do with your plugin)

anyway Im happy with your pluggin and this "request" is just a small thing

: )

**Reply by lukas-reineke** [reply]
You want to enter a literal tab character when you press the tab key in insert mode? That is a built-in setting, take a look at `:help 'expandtab'`. By default, it should input a tab, and not spaces.

**Reply by Belamat** [reply]
I think I got it working pretty well now. 

Have a nice day!
Thanks!!
